rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -376,7 +401,7 @@ public void run() {
         }
     }
 
-    private void restoreNodeSources() {
+    void restoreNodeSources() {
```

Comment:
For testing purposes. I have added the annotation `@VisibleForTesting` to make it clear.